frontend/src/App.js

```diff
@@ -22,6 +22,9 @@ import CoursesCreatePage from "main/pages/Courses/PSCourseCreatePage";
 
 import CourseOverTimeIndexPage from "main/pages/CourseOverTime/CourseOverTimeIndexPage";
 
+// PLACEHOLDER FOR PAGE
```

Comment:
I don't think we need this comment here.   The import won't change when this becomes a real page, but the comment is likely to stick around.  This is the "lie waiting to happen" I was referring to in the chat (I actually wrote that before I saw this, but it's a perfect example.)